_drafts/programatically-configured-bots.md

```diff
@@ -52,9 +52,9 @@ robot.alias = '/'
 # like robot name
 robot.adapter.once 'connected', ->
 
-  # require external hubot scripts
-  robot.loadScriptPackage require('hubot-help')
-  robot.loadScriptPackage require('hubot-shipit')
+  # require external hubot script packages
+  robot.loadPackage 'hubot-help'
+  robot.loadPackage 'hubot-shipit'
```

Comment:
I have definitely been thinking about how help works, as part of this, but also in general (see the #contributors discussion around hubot-better-help). I'm not sure I want to do it as this evolution though.